docs/index.md

```diff
@@ -295,20 +295,32 @@ If your aws credential file contains profiles that assume a role via the `source
 aws_access_key_id = AKIA4YFAKEKEYXTDS252
 aws_secret_access_key = SH42YMW5p3EThisIsNotRealzTiEUwXN8BOIOF5J8m
 region = us-west-2
+# this Principal must have permissions to sts:AssumeRole for 'arn:aws:iam::*:role/spc_role'
 
-[role-without-mfa]
-role_arn = arn:aws:iam::123456789012:role/test_assume
+[profile_account_a]
+role_arn = arn:aws:iam::111111111111:role/spc_role
 source_profile = cli-user
 external_id = xxxxx
+
+[profile_account_b]
+role_arn = arn:aws:iam::222222222222:role/spc_role
+source_profile = cli-user
+external_id = yyyyy
 ```
 
 #### aws.spc:
 
 ```hcl
-connection "role_aws" {
+connection "aws_account_a" {
   plugin  = "aws"
-  profile = "role-without-mfa"
-  regions = ["us-east-1", "us-east-2"]
+  profile = "profile_account_a"
+  regions = ["*"]
```

Comment:
```suggestion
  regions = ["us-east-1", "us-east-2"]
```

To diversify the regions/show different configs, what if we had this connection as an explicit list, while the connection below uses `*`?